src/components/SplashPage/SplashPage.js

```diff
@@ -1,16 +1,11 @@
 import Feed from '../Feed/Feed.js'
-import SearchBar from '../SearchBar/SearchBar.js'
-import SearchResults from '../SearchResults/SearchResults.js'
 import Header from '../Header/Header.js'
 
-const SplashPage = ({query, setQuery, queryResults, login}) => {
```

Comment:
Way to clean this up! This might get removed entirely for the feed page eventually, we'll just keep whittling it down